test/models/message/document_image_test.rb

```diff
@@ -16,12 +16,12 @@ class Message::DocumentImageTest < ActiveSupport::TestCase
     refute messages(:examine_this).has_document_image?(:small)
   end
 
-  test "document_image_path" do
-    assert messages(:examine_this).document_image_path(:small).is_a?(String)
-    assert messages(:examine_this).document_image_path(:small).starts_with?("/rails/active_storage/representations/redirect")
+  test "document_image_url" do
+    assert messages(:examine_this).document_image_url(:small).is_a?(String)
+    assert messages(:examine_this).document_image_url(:small).starts_with?("/rails/active_storage/representations/redirect")
   end
 
-  test "document_image_path with fallback" do
-    assert_equal "", messages(:examine_this).document_image_path(:small, fallback: "")
+  test "document_image_url with fallback" do
+    assert_equal "", messages(:examine_this).document_image_url(:small, fallback: "")
```

Comment:
If we can make the keys optional, then can we easily test both configurations with automated tests? I think so by using our stubbing technique for options.yml. But if that is tricky for some reason so that we can only have automated tests for one approach, then I think we should have the automated tests for the default config (i.e. all url_* being blank)